target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -675,6 +676,10 @@ ramips_setup_macs()
 	iptime,a604m)
 		wan_mac=$(mtd_get_mac_binary u-boot 0x1fc40)
 		;;
+	iptime,a8004t)
+		lan_mac=$(mtd_get_mac_binary u-boot 0x1fc20)
```

Comment:
You should set this via &ethernet in the DTS, just as done for the other iptime devices:
https://github.com/openwrt/openwrt/commit/b7ec6659a14d1a11e52b56235274b3e46d22e331#diff-48c2cd5429a0b46871e6198b9bd61f8cR90
Then, remove lan_mac here (as it will be inherited from eth0), but keep wan_mac.

Reply:
Ok, fixed.